src/els_indexer.erl

```diff
@@ -229,3 +181,30 @@ register_reference(Uri, #{id := {M, F, A}, range := Range}) ->
                             , uri   => Uri
                             , range => Range
                             }).
+
+-spec index_dir(string(), mode()) -> {non_neg_integer(), non_neg_integer()}.
+index_dir(Dir, Mode) ->
+  lager:info("Indexing directory. [dir=~s] [mode=~s]", [Dir, Mode]),
```

Comment:
Could we change this and the "Finished indexing..." log messages from `info` to `debug`? If they are at the `info` level it makes the logs very verbose by default.

Reply:
Yes, I can do that.

Reply:
Fixed.